feature-pack/src/main/resources/modules/system/layers/base/org/jboss/jts/main/module.xml

```diff
@@ -47,5 +47,11 @@
         <module name="org.jboss.weld.core"/>
         <module name="javax.annotation.api" export="true" />
         <module name="javax.interceptor.api" export="true" />
+        <module name="org.wildfly.transaction.client"/>
+        <module name="org.jboss.as.transactions">
+            <imports>
+                <include path="org.jboss.as.txn.integration"/>
```

Comment:
that would not be mixing in one file but over transaction related files. I think the dot notation is used in modules `jts`,`webservices` (conected to xts), `transaction integration`. I could change all of them to `/` for not mixing. In fact I used `.` as I was inspired by `transaction integration` module.